### PR TITLE
Add OboResourceAccountIds field to New-CsCallQueue and Set-CsCallQueue

### PR DESCRIPTION
**Pull request by andyqiu-microsoft**
Add OboResourceAccountIds field to New-CsCallQueue and Set-CsCallQueue.

### CODE REVIEW

File: skype/skype-ps/skype/New-CsCallQueue.md [hunk 1]
```diff
@@ -129,6 +129,24 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -OboResourceAccountIds
+The OboResourceAccountIds parameter lets you add resource account with phone number to the Call Queue. The agents in the Call Queue will be able to make outbound calls using the phone number on the resource accounts.  This is a list of resource account GUIDs.
+
+Only Call Queue managed by a Teams Channel will be able to use this feature. For more information, please refer to [Manage your support Call Queue in Teams](https://support.microsoft.com/en-us/office/manage-your-support-call-queue-in-teams-9f07dabe-91c6-4a9b-a545-8ffdddd2504e).
+
+```yaml
+Type: List
```
Comment (dariomws) [inline skype/skype-ps/skype/New-CsCallQueue.md:138]:
```suggestion
Type: Guid
```
It seems that this parameter supports a single GUID as a value:
image

File: skype/skype-ps/skype/Set-CsCallQueue.md [hunk 2]
```diff
@@ -143,6 +143,24 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -OboResourceAccountIds
+The OboResourceAccountIds parameter lets you add resource account with phone number to the Call Queue. The agents in the Call Queue will be able to make outbound calls using the phone number on the resource accounts.  This is a list of resource account GUIDs.
+
+Only Call Queue managed by a Teams Channel will be able to use this feature. For more information, please refer to [Manage your support Call Queue in Teams](https://support.microsoft.com/en-us/office/manage-your-support-call-queue-in-teams-9f07dabe-91c6-4a9b-a545-8ffdddd2504e).
+
+```yaml
+Type: List
```
Comment (dariomws) [inline skype/skype-ps/skype/Set-CsCallQueue.md:152]:
```suggestion
Type: Guid
```

File: skype/skype-ps/skype/New-CsCallQueue.md [hunk 3]
```diff
@@ -22,7 +22,7 @@ New-CsCallQueue -Name <String> [-AgentAlertTime <Int16>] [-AllowOptOut <Boolean>
...
 [-OverflowAction <Object>] [-OverflowActionTarget <Guid>] [-OverflowThreshold <Int16>] 
 [-TimeoutAction <Object>] [-TimeoutActionTarget <Guid>] [-TimeoutThreshold <Int16>] 
-[-RoutingMethod <Object>] [-PresenceBasedRouting <Boolean>] [-ConferenceMode <Boolean>] [-User <List>] [-LanguageId <String>] [-LineUri <String>] [-OverflowSharedVoicemailTextToSpeechPrompt <String>] [-OverflowSharedVoicemailAudioFilePrompt <Guid>] [-EnableOverflowSharedVoicemailTranscription <Boolean>] [-TimeoutSharedVoicemailTextToSpeechPrompt <String>] [-TimeoutSharedVoicemailAudioFilePrompt <Guid>] [-EnableTimeoutSharedVoicemailTranscription <Boolean>] 
+[-RoutingMethod <Object>] [-PresenceBasedRouting <Boolean>] [-ConferenceMode <Boolean>] [-User <List>] [-LanguageId <String>] [-LineUri <String>] [-OboResourceAccountIds <List>] [-OverflowSharedVoicemailTextToSpeechPrompt <String>] [-OverflowSharedVoicemailAudioFilePrompt <Guid>] [-EnableOverflowSharedVoicemailTranscription <Boolean>] [-TimeoutSharedVoicemailTextToSpeechPrompt <String>] [-TimeoutSharedVoicemailAudioFilePrompt <Guid>] [-EnableTimeoutSharedVoicemailTranscription <Boolean>]
```
Comment (dariomws) [inline skype/skype-ps/skype/New-CsCallQueue.md:25]:
```suggestion
[-RoutingMethod <Object>] [-PresenceBasedRouting <Boolean>] [-ConferenceMode <Boolean>] [-User <List>] [-LanguageId <String>] [-LineUri <String>] [-OboResourceAccountIds <Guid>] [-OverflowSharedVoicemailTextToSpeechPrompt <String>] [-OverflowSharedVoicemailAudioFilePrompt <Guid>] [-EnableOverflowSharedVoicemailTranscription <Boolean>] [-TimeoutSharedVoicemailTextToSpeechPrompt <String>] [-TimeoutSharedVoicemailAudioFilePrompt <Guid>] [-EnableTimeoutSharedVoicemailTranscription <Boolean>] 
```

File: skype/skype-ps/skype/New-CsCallQueue.md [hunk 4]
```diff
@@ -129,6 +129,24 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -OboResourceAccountIds
+The OboResourceAccountIds parameter lets you add resource account with phone number to the Call Queue. The agents in the Call Queue will be able to make outbound calls using the phone number on the resource accounts.  This is a list of resource account GUIDs.
```
Comment (dariomws) [inline skype/skype-ps/skype/New-CsCallQueue.md:133]:
```suggestion
The OboResourceAccountIds parameter lets you add a resource account with a phone number to the Call Queue. The agents in the Call Queue will be able to make outbound calls using the phone number on the resource accounts. This is a Guid which should contain 32 digits with 4 dashes (xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx).
```

File: skype/skype-ps/skype/Set-CsCallQueue.md [hunk 5]
```diff
@@ -18,7 +18,7 @@ Updates a Call Queue in your Skype for Business Online or Teams organization.
...
+Set-CsCallQueue -Identity <Guid> [-AgentAlertTime <Int16>] [-AllowOptOut <Boolean>] [-DistributionLists <List>] [-MusicOnHoldAudioFileId <Guid>] [-Name <String>] [-OboResourceAccountIds <List>] [-OverflowAction <Object>] [-OverflowActionTarget <Guid>] [-OverflowThreshold <Int16>] [-RoutingMethod <Object>] [-TimeoutAction <Object>] [-Tenant <Guid>] [-TimeoutActionTarget <Guid>] [-TimeoutThreshold <Int16>] [-UseDefaultMusicOnHold <Boolean>] [-WelcomeMusicAudioFileId <Guid>] [-PresenceBasedRouting <Boolean>] [-ConferenceMode <Boolean>] [-User <List>] [-LanguageId <String>] [-LineUri <String>] [-OverflowSharedVoicemailTextToSpeechPrompt <String>] [-OverflowSharedVoicemailAudioFilePrompt <Guid>] [-EnableOverflowSharedVoicemailTranscription <Boolean>] [-TimeoutSharedVoicemailTextToSpeechPrompt <String>] [-TimeoutSharedVoicemailAudioFilePrompt <Guid>] [-EnableTimeoutSharedVoicemailTranscription <Boolean>] [<CommonParameters>]
```
Comment (dariomws) [inline skype/skype-ps/skype/Set-CsCallQueue.md:21]:
```suggestion
Set-CsCallQueue -Identity <Guid> [-AgentAlertTime <Int16>] [-AllowOptOut <Boolean>] [-DistributionLists <List>] [-MusicOnHoldAudioFileId <Guid>] [-Name <String>] [-OboResourceAccountIds <Guid>] [-OverflowAction <Object>] [-OverflowActionTarget <Guid>] [-OverflowThreshold <Int16>] [-RoutingMethod <Object>] [-TimeoutAction <Object>] [-Tenant <Guid>] [-TimeoutActionTarget <Guid>] [-TimeoutThreshold <Int16>] [-UseDefaultMusicOnHold <Boolean>] [-WelcomeMusicAudioFileId <Guid>] [-PresenceBasedRouting <Boolean>] [-ConferenceMode <Boolean>] [-User <List>] [-LanguageId <String>] [-LineUri <String>] [-OverflowSharedVoicemailTextToSpeechPrompt <String>] [-OverflowSharedVoicemailAudioFilePrompt <Guid>] [-EnableOverflowSharedVoicemailTranscription <Boolean>] [-TimeoutSharedVoicemailTextToSpeechPrompt <String>] [-TimeoutSharedVoicemailAudioFilePrompt <Guid>] [-EnableTimeoutSharedVoicemailTranscription <Boolean>] [<CommonParameters>]
```

File: skype/skype-ps/skype/Set-CsCallQueue.md [hunk 6]
```diff
@@ -143,6 +143,24 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -OboResourceAccountIds
+The OboResourceAccountIds parameter lets you add resource account with phone number to the Call Queue. The agents in the Call Queue will be able to make outbound calls using the phone number on the resource accounts.  This is a list of resource account GUIDs.
```
Comment (dariomws) [inline skype/skype-ps/skype/Set-CsCallQueue.md:147]:
```suggestion
The OboResourceAccountIds parameter lets you add a resource account with a phone number to the Call Queue. The agents in the Call Queue will be able to make outbound calls using the phone number on the resource accounts. This is a Guid which should contain 32 digits with 4 dashes (xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx).
```

**Review by dariomws — CHANGES_REQUESTED** [review]

**Comment by dariomws** [comment]
If you have a Microsoft alias, you can link your accounts in GitHub so that a blue "Microsoft submitter" label appears to any PR's or issue you submit. Guidance on it here: https://docs.opensource.microsoft.com/tools/github/index.html

Here is the site you can use to see the mappings between GitHub and Microsoft aliases: https://repos.opensource.microsoft.com/people?q=andyqiu-microsoft

File: skype/skype-ps/skype/New-CsCallQueue.md (hunk 1 above)
Comment (andyqiu-microsoft) [reply skype/skype-ps/skype/New-CsCallQueue.md:138]: This parameter accept a single GUID and a list of GUID, so the parameter name has a "s" after it.  Should the type be Guid or List in this case? Please see the following for example:
image
image

File: skype/skype-ps/skype/New-CsCallQueue.md [hunk 7]
```diff
@@ -129,6 +129,24 @@ Accept pipeline input: False
...
+
+Only Call Queue managed by a Teams Channel will be able to use this feature. For more information, please refer to [Manage your support Call Queue in Teams](https://support.microsoft.com/en-us/office/manage-your-support-call-queue-in-teams-9f07dabe-91c6-4a9b-a545-8ffdddd2504e).
```
[the remark this answers is not in the document]
Comment (andyqiu-microsoft) [reply skype/skype-ps/skype/New-CsCallQueue.md]: The link has been updated.  Thanks for suggestion.

File: skype/skype-ps/skype/Set-CsCallQueue.md [hunk 8]
```diff
@@ -143,6 +143,24 @@ Accept pipeline input: False
...
+
+Only Call Queue managed by a Teams Channel will be able to use this feature. For more information, please refer to [Manage your support Call Queue in Teams](https://support.microsoft.com/en-us/office/manage-your-support-call-queue-in-teams-9f07dabe-91c6-4a9b-a545-8ffdddd2504e).
```
[the remark this answers is not in the document]
Comment (andyqiu-microsoft) [reply skype/skype-ps/skype/Set-CsCallQueue.md]: The link has been updated.  Thanks for suggestion.

File: skype/skype-ps/skype/New-CsCallQueue.md (hunk 1 above)
Comment (dariomws) [reply skype/skype-ps/skype/New-CsCallQueue.md:138]: Agree, my mistake. Let's keep List. Thank you!

**Review by dariomws — APPROVED** [review]

**Comment by dariomws** [comment]
> If you have a Microsoft alias, you can link your accounts in GitHub so that a blue "Microsoft submitter" label appears to any PR's or issue you submit. Guidance on it here: https://docs.opensource.microsoft.com/tools/github/index.html
> 
> Here is the site you can use to see the mappings between GitHub and Microsoft aliases: https://repos.opensource.microsoft.com/people?q=andyqiu-microsoft

Please don't forget to map your accounts, so a Microsoft Submitter label is added next PR. Thanks!

File: skype/skype-ps/skype/New-CsCallQueue.md (hunk 1 above)
Comment (andyqiu-microsoft) [reply skype/skype-ps/skype/New-CsCallQueue.md:138]: Thanks!  I will resolve this comment and the related ones.

**Review by get-itips — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tseward

**Review by tseward — APPROVED** [review]

**Comment by andyqiu-microsoft** [comment]
@dariomws Is there any extra steps before this PR can be merged?

**Comment by dariomws** [comment]
No, it should be merged soon. Thank you!

@Kateyanne

**Comment by yogkumgit** [comment]
@Kateyanne Please merge this PR. Thanks

**Comment by DEberhardt** [comment]
Don't know how to resolve "This branch is out-of-date with the base branch" - may I have some advice on how to proceed here?

**Comment by andyqiu-microsoft** [comment]
I have resolve the "This branch is out-of-date with the base branch" warning. Thanks for the comment.

**Comment by garycentric** [comment]
@andyqiu-microsoft

Please resolve the merge conflict. When this PR is ready for review, please remove _Back to submitter_ and reapply _Sign off_.

Thanks,

Gary

**Comment by andyqiu-microsoft** [comment]
@garycentric 

I don't think I have the permission to change the label of this PR.  All merge conflict of this PR has been resolved.

Thanks,
Andy

**Comment by garycentric** [comment]
@Kateyanne

I was going to make the simple change to this PR that I've made in others for PowerShell content—remove the long lines of hyphens from the H3 "Example" headings—but I'm prevented from pushing changes to the branch from my computer and from selecting **Edit file** on the **Files changed** tab in GitHub. I'm going to merge this PR as it is, because this isn't a critical change, but please let me know if this block to my edits is expected.

**Pull request merged**